API.md

```diff
@@ -9,7 +9,7 @@
 ## Functions
 
 <dl>
-<dt><a href="#once">once(id, elements)</a> ⇒ <code>Array.&lt;Element&gt;</code></dt>
+<dt><a href="#once">once(id, input, [context])</a> ⇒ <code>Array.&lt;Element&gt;</code></dt>
```

Comment:
any better naming than `input`? `target`? `elementQuery|queryElements` ?

Reply:
jQuery uses 'selector' for this type of thing.

Reply:
`selector` is good.